src/main/java/de/teamlapen/vampirism/entity/minion/MinionEntity.java

```diff
@@ -602,6 +602,7 @@ private void handleLoadedMinionData(@NotNull T data) {
         this.getAttribute(Attributes.MAX_HEALTH).setBaseValue(data.getMaxHealth());
         super.setHealth(data.getHealth());
         super.setCustomName(data.getFormattedName());
+        this.deserializeCaps(data.getEntityCaps());
```

Comment:
Saving an loading all available caps may introduce side effects that we do not want. E.g. if a cap kills the minion, loading the cap may constantly kill them again.